search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java

```diff
@@ -436,9 +441,16 @@ void searchUriWithSortingReturnsOpenSearchAwsResponse(URI uri) throws ApiGateway
             var pagedSearchResourceDto = response.toPagedResponse();
             assertNotNull(pagedSearchResourceDto.id());
             var searchName = response.parameters().get(SORT).split(COMMA)[0].split(COLON)[0];
-            var searchFieldName = ResourceSort.fromSortKey(searchName).jsonPaths().findFirst().get();
+            var searchFieldName = ResourceSort.fromSortKey(searchName)
```

Comment:
so now you always need a sort key or else it throws error?

Reply:
Sort is always needed, that is why it has default values that are applied, when no value are provided by api user.